commands.json

```diff
@@ -1,11 +1,31 @@
+    "?openballs": "Couple weeks into calculus 3 now, doing well, already past open sets and continuity in R^d. Convexity was a joke. Open balls remain my specialty. I ask my professor his thoughts on string theory and “monster” groups. He's impressed I know about the subject. We converse after class for some time, sharing mathematical insights; I can keep up. He tells me of great things ahead like differential equations and the Jacobian. I tell him I already read about differential equations in physics class. He is yet again impressed at my enthusiasm. What a joy it is to have your professor visibly brighten when he learns of your talents. And now I sit here wondering what it must be like to be a brainlet, unable to engage your professor as an intellectual peer. All of the deep conversations you people must miss out on because you aren't able to overcome the intellectual IQ barrier that stands in the way of your academic success... it's so sad. My professor and I know each other on first name basis now, but i call him Dr. out of respect. And yet here you brainlets sit, probably havent even made eye contact with yours out of fear that they will gauge your brainlet IQ levels. A true shame, but just know it is because i was born special that i am special. I can't help being a genius, nor can my professor. Two of a kind is two flocks in a bush.",
```

Comment:
change the quotes around monster to normal double quotes. The ones currently is too fancy and I keep getting a unicode decode error on my end. 
```suggestion
    "?openballs": "Couple weeks into calculus 3 now, doing well, already past open sets and continuity in R^d. Convexity was a joke. Open balls remain my specialty. I ask my professor his thoughts on string theory and \"monster\" groups. He's impressed I know about the subject. We converse after class for some time, sharing mathematical insights; I can keep up. He tells me of great things ahead like differential equations and the Jacobian. I tell him I already read about differential equations in physics class. He is yet again impressed at my enthusiasm. What a joy it is to have your professor visibly brighten when he learns of your talents. And now I sit here wondering what it must be like to be a brainlet, unable to engage your professor as an intellectual peer. All of the deep conversations you people must miss out on because you aren't able to overcome the intellectual IQ barrier that stands in the way of your academic success... it's so sad. My professor and I know each other on first name basis now, but i call him Dr. out of respect. And yet here you brainlets sit, probably havent even made eye contact with yours out of fear that they will gauge your brainlet IQ levels. A true shame, but just know it is because i was born special that i am special. I can't help being a genius, nor can my professor. Two of a kind is two flocks in a bush.",
```